bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -236,7 +236,8 @@ public void visit(WhitespaceIssue e) {
     @Override
     public void visit(MessageIssue issue) {
         var message = String.join("\n", issue.commit().message());
-        throw new IllegalStateException("Incorrectly formatted commit message: " + message);
+        log.info("Incorrectly formatted commit message: " + message);
```

Comment:
This warrants a warning level message I think. We really don't expect this to happen.